bundle/jsky.app.ot/src/main/java/jsky/app/ot/gemini/editor/EdProgram.java

```diff
@@ -244,7 +244,9 @@ private static void setTimeLabel(final long time, final JLabel timeLabel) {
         timeLabel.setText(timeStr);
     }
     private static void setTimeLabel(final double time, final JLabel timeLabel) {
-        final String timeStr = time == 0d ? ZERO_TIME_STRING : new HMS(time).toString();
+        final String timeStr = time == 0d ?
+                ZERO_TIME_STRING :
+                new HMS(time).toString(true, false);
```

Comment:
Actually, since you had to modify the old `jsky` `HMS`, maybe it would be better to just turn `time` into milliseconds and format with the `java.time` API.